library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -225,7 +225,7 @@ public void setOnCameraChangeListener(final OnCameraChangeListener onCameraChang
         if (airMarker != null) {
           listener.onMapMarkerClick(airMarker);
         }
-        return false;
+        return listener.onMakerClickEventIsHandled();
```

Comment:
Why not add a return value to `onMapMarkerClick` instead? Seems more idiomatic.

Reply:
I pushed a change that addressed this comment